apps/prairielearn/src/ee/pages/administratorInstitutionAdmins/administratorInstitutionAdmins.html.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { Institution } from '../../../lib/db-types.js';
+
+export function AdministratorInstitutionAdmins({
+  institution,
+  resLocals,
+}: {
+  institution: Institution;
+  resLocals: Record<string, any>;
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/head')%>", {
+          ...resLocals,
+          navPage: 'administrator_institution',
+          pageTitle: 'Admins',
+        })}
+      </head>
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/navbar') %>", {
+          ...resLocals,
+          institution,
+          navbarType: 'administrator_institution',
+          navPage: 'administrator_institution',
+          navSubPage: 'admins',
+        })}
+        <main class="container mb-4">
+          <p>
+            Institution administrators can be managed
+            <a href="/pl/institution/${institution.id}/admin/admins">here</a>.
```

Comment:
I'm lazy and allergic to code duplication, so this is a compromise that shows us a tab where we expect it to be (in the global admin institution pages) while not duplicating the entire page here.

Reply:
See my suggestion elsewhere about unifying the two hierarchies (`institutionAdmin*` and `administratorInstitution*`) as another way to address this.

Reply:
Discussed with @mwest1066, we're going to retain the two separate hierarchies for now.